bin/migration-scripts/lts-to-lts/0.16-to-0.23/README.md

```diff
@@ -0,0 +1,96 @@
+# Astronomer upgrade guide
+
+The Astronomer upgrade will run in a pod in the cluster.
+
+# Before you upgrade
+
+## Check version compatibility
+
+- Kubernetes version must be greater than or equal to 1.14 and less than 1.19. If you need help to upgrade, please contact your cloud provider's support or your Kubernetes administrator.
+- You must be using Astronomer Certified Airflow images, and your version must be 1.10.5 or greater. These images are in the form:
+```
+astronomerinc/ap-airflow:<airflow-version>-<build number>-<distribution>-onbuild
+```
+The "onbuild" part is optional, but recommended. This enables features in the astro-cli, such as using requirements.txt and packages.txt. The "build number" part is optional but recommended. Excluding the build number will referene a mutable Docker image that is occaisonally replaced with underlying pip modules updated, this is typically not preferred and instead it's best to plan when updates will occur in your Docker image. If you have your own build, test, and publish workflows that are layered on top of the Astronomer Airflow images, then it is appropriate to use the mutable image.
+
+Valid examples:
+```
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10-onbuild
+astronomerinc/ap-airflow:1.10.12-1-alpine3.10
+astronomerinc/ap-airflow:1.10.5-9-buster-onbuild
+astronomerinc/ap-airflow:1.10.5-9-buster
+```
+
+This is an example of a legacy images that should *not* be used:
+```
+astronomerinc/ap-airflow:<astronomer version>-<airflow version>
+astronomerinc/ap-airflow:
+```
+
+## Check permissions
+
+You should be an Astronomer admin to perform the upgrade. This can be confirmed if you have access Grafana: https://grafana.your-base-domain.com or by checking that you have access to the "system admin" feature in Astronomer UI.
+
+You will be creating Kubernetes resources to perform this upgrade. Kubernetes permission can be validated with:
+
+```
+kubectl auth can-i create pods --namespace <your astronomer namespace>
+kubectl auth can-i create sa --namespace <your astronomer namespace>
+kubectl auth can-i create jobs --namespace <your astronomer namespace>
+```
+
+All lines should say "yes".
+
+## Take an external DB backup
+
+Backup your database using your cloud provider's tool or make a request to your database administrator.
+
+## Check on the status of your pods
+
+All pods should be "Running" or "Completed". If you have any pods that are crashing but this is expected for some reason and you want to proceed anyways, then make note of which pods are crashing before the upgrade has been performed.
+
```

Comment:
@sjmiller609 should we add a section recommending a velero backup as well?

Reply:
yeah, I think that's a good call